src/deploy/schema.json

```diff
@@ -40,19 +48,34 @@
     },
     "noSilent": {
       "type": "boolean",
-      "description": "Logging is in silent mode by default. Execute with --no-silent to enable extended console logging. Keep this untouched if the repository URL or other information passed to git commands is sensitive!",
+      "description": "Deprecated! This paremeter is no longer needed. It will be ignored.",
       "default": false
     },
     "noDotfiles": {
       "type": "boolean",
       "description": "Includes dotfiles by default. Execute with --no-dotfiles to ignore files starting with `.`.",
       "default": false
     },
+    "noNotfound": {
+      "type": "boolean",
+      "description": "By default a 404.html file is created, because this is the only known workaround to avoid 404 error messages on GitHub pages. For Cloudflare Pages we highly recommend to disable the 404.html file by setting this switch to true!",
```

Comment:
```suggestion
      "description": "By default a 404.html file is created, because this is the only known workaround to avoid 404 error messages on GitHub Pages. For Cloudflare Pages we highly recommend to disable the 404.html file by setting this switch to true!",
```